scripts/remove

```diff
@@ -100,8 +83,19 @@ systemctl daemon-reload
 ynh_print_info "Removing dependencies"
 ynh_remove_app_dependencies
 
-# Remove sources
-ynh_secure_remove "/var/www/${app}"
+#=================================================
+# REMOVE LEGACY INTERFACES
+#=================================================
+if [ -d /etc/nginx/conf.d/*.d/$app.conf ]; then
+	ynh_secure_remove /etc/nginx/conf.d/*.d/$app.conf
+fi
+if [ -d /etc/php/*/fpm/pool.d/$app.conf ]; then
+	ynh_secure_remove /etc/php/*/fpm/pool.d/$app.conf
+fi
```

Comment:
```suggestion
if [ -f /etc/nginx/conf.d/*.d/$app.conf ]; then
	ynh_secure_remove /etc/nginx/conf.d/*.d/$app.conf
fi
if [ -f /etc/php/*/fpm/pool.d/$app.conf ]; then
	ynh_secure_remove /etc/php/*/fpm/pool.d/$app.conf
fi
```
I think the issue I mentionned just above is because you check if the config file is a directory (instead of a regular file)

Reply:
Also, you should reload nginx and php-fpm after these files have been removed.

Reply:
Ah actually the issue lies in the upgrade script but I will make a bunch of followup commits